qemu/tests/cfg/nvdimm.cfg

```diff
@@ -92,3 +92,17 @@
         - nvdimm_hotplug:
             del mem_devs
             target_mems = mem1
+        - nvdimm_negative:
+            type = nvdimm_negative
+            start_vm = no
+            size_mem = 8G
+            pre_command = "dd if=/dev/zero of=${nv_backend} bs=1M count=8192"
+            variants:
+                - align_max:
+                    align_mem = 4G
+                    error_msg = "the alignment \(0x100000000\) is not supported|the alignment \(0x100000000\) exceeds the expected maximum alignment"
```

Comment:
On ppc, "-device nvdimm" also needs "uuid" parameter.
And according the current parameters
size = 8G
lable_size=256M
align_mem =4G
The error output should be "backing store size 0x80000000 does not match 'size' option 0x200000000
" on ppc

Reply:
Does every case need 'uuid' for ppc? What's the expected value for uuid?
Besides, pls update the error message to polarion,  it says no error for ppc.

Reply:
On ppc, the combination paramters are 2G,2G and 128K in the test case but now is 8G, 4G and 256M. but I can accept them, please get uuid by uuidgen,and polarion is done as well.